packages/create/src/index.ts

```diff
@@ -149,14 +162,18 @@ export function defaultPeerDiscovery(): PeerDiscovery {
 
 export async function defaultLibp2p(
   wakuRelay?: WakuRelay,
-  options?: Partial<Libp2pOptions>
+  options?: Partial<Libp2pOptions>,
+  userAgent?: string
 ): Promise<Libp2p> {
   const libp2pOpts = Object.assign(
     {
       transports: [new WebSockets({ filter: filterAll })],
       streamMuxers: [new Mplex()],
       connectionEncryption: [new Noise()],
-    },
+      identify: {
+        host: userAgent || DefaultUserAgent,
```

Comment:
```suggestion
        host: userAgent ?? DefaultUserAgent,
```

If you use `||` then empty string `0` and number value `0` will be overridden by the default user agent which is not the behaviour we'd expect AFAIK.